awscli/examples/eks/wait/addon-active.rst

```diff
@@ -0,0 +1,11 @@
+**To wait for an add-on running in the Amazon EKS cluster to become ACTIVE**
+
+The following ``wait addon-active`` example command waits for an add-on named ``aws-efs-csi-driver`` running in the Amazon EKS cluster named ``my-eks-cluster`` to become ``ACTIVE``. ::
+
+    aws eks wait addon-active \
+        --cluster-name my-eks-cluster \
+        --addon-name aws-efs-csi-driver
+
+Output::
+
+    Waiter AddonActive failed: Waiter encountered a terminal failure state: For expression "addon.status" we matched expected path: "DEGRADED"
```

Comment:
Ideally, the example outputs should follow the happy path and not show a failure.